UnRen.bat

```diff
@@ -148,27 +148,27 @@ if exist "%rpatool%" (
 	del "%rpatool%"
 )
 
-:: echo %rpatool%>> "%rpatool%.tmp"
+REM echo %rpatool%>> "%rpatool%.tmp"
 [RPATOOL//MERGE]
 set "rpatoolps=%rpatool:[=`[%"
 set "rpatoolps=%rpatoolps:]=`]%"
 set "rpatoolps=%rpatoolps:^=^^%"
 set "rpatoolps=%rpatoolps:&=^&%"
-powershell.exe -nologo -noprofile -noninteractive -command "& { [IO.File]::WriteAllBytes(\"%rpatoolps%\", [Convert]::FromBase64String([IO.File]::ReadAllText(\"%rpatoolps%.tmp\"))) }"
+powershell.exe -nologo -noprofile -noninteractive -command "& { [IO.File]REMWriteAllBytes(\"%rpatoolps%\", [Convert]REMFromBase64String([IO.File]REMReadAllText(\"%rpatoolps%.tmp\"))) }"

```

Comment:
Mistaken `::` -> `REM` replace

Reply:
Oh no. I checked... and feel now blind. *grumbles* I have no windows with powershell available at the moment, so i cannot test.